src/UI/Components/Diagram/init.ts

```diff
@@ -360,16 +82,21 @@ export default function diagramInit(
     },
 
     addInstance: (
-      instance: InstanceWithRelations,
       services: ServiceModel[],
-      isMainInstance: boolean,
+      instance?: InstanceWithRelations,
     ) => {
-      appendInstance(paper, graph, instance, services, isMainInstance);
+      if (!instance) {
+        populateGraph(graph, mainService);
+      } else {
+        const isMainInstance = true;
+        appendInstance(paper, graph, instance, services, isMainInstance);
```

Comment:
isMainInstance is by default set to true, you could technically omit the argument entirely